vsphere/resource_vsphere_virtual_machine_test.go

```diff
@@ -836,6 +836,45 @@ func TestAccResourceVSphereVirtualMachine(t *testing.T) {
 				},
 			},
 		},
+		{
+			"clone with vapp properties",
```

Comment:
Some thoughts on testing this while enforcing present keys only - we might need to test this against a CoreOS template or something else well known, and work on setting well-known cloud-config keys.

Reply:
I've started testing against the CoreOS OVA instead of against a custom-built template.